### PR TITLE
Cluster statistics

### PR DESCRIPTION
**Pull request by mkabischev**
Hi,

I've added statistics to regido's pool: https://github.com/garyburd/redigo/pull/282. This PR allows getting statistics for all pools in the cluster.

### CODE REVIEW

File: cluster.go [hunk 1]
```diff
@@ -391,3 +391,17 @@ func (c *Cluster) Close() error {
 
 	return err
 }
+
+// Stats returns the current statistics for all pools.
```
Comment (mna) [inline cluster.go]: Could you add a line indicating that the statistics are keyed by the cluster node's address?

**Comment by mna** [comment]
Thanks! Usually I would've asked for tests but this is a straightforward and short function call. I just added a comment to add a line for the godoc, otherwise it LGTM. I'll merge once the comment is addressed.

Martin

**Comment by mkabischev** [comment]
@mna I've added the comment. 
Thanks for fast reaction.

**Comment by mna** [comment]
Thanks for the PR!

**Pull request merged**